dev-resources/logback-test.xml

```diff
@@ -5,10 +5,6 @@
         </encoder>
     </appender>
 
-    <logger name="puppetlabs" level="trace"/>
-    <logger name="log4j.logger.org.eclipse.jetty.server" level="warn"/>
-    <logger name="log4j.logger.org.eclipse.jetty.util.log" level="warn"/>
-
     <root level="error">
```

Comment:
I can make that `warn`, I originally just copied it from the
`dev-resources/logback.xml`. Should I update that while I'm here as well?
We seem pretty consistent on `logback-test.xml` being set to  `warn`, but
we seem inconsistent about having a normal `logback.xml` or
`logback-dev.xml` in ether `dev` or `dev-resourse` with either `warn` or
`debug` levels (comparing puppetserver, tk, tk-auth, tk-status,
pe-file-sync)

On Tue, May 3, 2016 at 11:10 AM, Kevin Corcoran <notifications@github.com>
wrote:

> In dev-resources/logback-test.xml
> <https://github.com/puppetlabs/ring-middleware/pull/28#discussion_r61928142>
> :
>
> > @@ -5,10 +5,6 @@
> >          </encoder>
> >      </appender>
> >
> > -    <logger name="puppetlabs" level="trace"/>
> > -    <logger name="log4j.logger.org.eclipse.jetty.server" level="warn"/>
> > -    <logger name="log4j.logger.org.eclipse.jetty.util.log" level="warn"/>
> > -
> >      <root level="error">
>
> This is warn instead of error in most (all?) of our other projects.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/puppetlabs/ring-middleware/pull/28/files/a24398d296eef6fcc4a2ca10e92b6fa1bece761f..ab694a9a5a3e45fae13c1b2d4471fbaa5909365c#r61928142>
>